specification/resources/resource-manager/Microsoft.Resources/stable/2019-11-01/subscriptions.json

```diff
@@ -0,0 +1,521 @@
+          "type": "string",
+          "description": "The physical location of the Azure location."
+        },
+        "pairedRegion": {
+          "type": "array",
+          "description": "The regions paired to this region.",
+          "items": {
+            "$ref": "#/definitions/PairedRegion"
+          }
+        }
+      }
+    },
+    "Location": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID of the location. For example, /subscriptions/00000000-0000-0000-0000-000000000000/locations/westus."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location name."
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location."
+        },
+        "regionalDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the location and its region."
+        },
+        "metadata": {
+          "description": "Metadata of the location, such as lat/long, paired region, and others.",
+          "$ref": "#/definitions/LocationMetadata"
+        }
+      },
+      "description": "Location information."
+    },
+    "LocationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Location"
+          },
+          "description": "An array of locations."
+        }
+      },
+      "description": "Location list operation response."
+    },
+    "Subscription": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID for the subscription. For example, /subscriptions/00000000-0000-0000-0000-000000000000."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription display name."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription tenant ID."
+        },
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription state. Possible values are Enabled, Warned, PastDue, Disabled, and Deleted.",
+          "enum": [
+            "Enabled",
+            "Warned",
+            "PastDue",
+            "Disabled",
+            "Deleted"
+          ],
+          "x-ms-enum": {
+            "name": "SubscriptionState",
+            "modelAsString": false
```

Comment:
why modelAsString false? it will cause breaking change in SDK for future updating

Reply:
That is how it was from the api-version I copied, I think it hasn't been updated since (2019-06-01)